tempesta_fw/connection.h

```diff
@@ -185,7 +185,8 @@ enum {
 	TFW_CONN_B_QFORWD,	/* Need to forward requests in the queue. */
 	TFW_CONN_B_HASNIP,	/* Has non-idempotent requests. */
 
-	TFW_CONN_B_DEL		/* Remove connection */
+	TFW_CONN_B_DEL,		/* Remove connection */
+	TFW_CONN_B_ACTIVE	/* Connection has been got at least once. */
```

Comment:
`Connection has been got at least once.` is not actually true. The flag means that the connection itself got reference to server. And it's not about of  connection's reference counter. `Connection was in use or at least scheduled to be established.`

Reply:
Corrected.